src/extension/coqPilot.ts

```diff
@@ -57,6 +58,7 @@ export class CoqPilot {
     private readonly globalExtensionState: GlobalExtensionState;
     private readonly vscodeExtensionContext: ExtensionContext;
     private subscriptions: { dispose(): any }[] = [];
+    private abortController = new AbortController();
```

Comment:
We should create an `AbortController` instance per user request, not keep a global one?

Reply:
Does re-creating an abort controller for each session bring any benefit? I thought of re-creating it only when the the last one was aborted and therefore invalidated?